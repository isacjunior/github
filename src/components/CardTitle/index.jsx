import React from 'react'
import PropTypes from 'prop-types'
import CardTitleStyled from './styled'

const CardTitle = ({ title }) => (
  <CardTitleStyled>
    <h2>{title}</h2>
  </CardTitleStyled>
)

CardTitle.proptypes = {
  title: PropTypes.string.isRequired
}

export default CardTitle