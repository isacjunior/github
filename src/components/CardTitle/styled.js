import styled from 'styled-components'

const CardTitleStyled = styled.div`
  padding-top: 30px;

  h2 {
    color: #FF6A67
  }

  .line-under {
    margin-top: 15px;
    height: 3px;
    width: 40px;
    background: #FF6A67;
    margin-bottom: 30px;
  }
`

export default CardTitleStyled
