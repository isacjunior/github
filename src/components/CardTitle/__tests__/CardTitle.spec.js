import React from 'react'
import { mount } from 'enzyme'
import CardTitle from '../index'

describe('CardTitle', () => {
  it('CardTitle render', () => {
    mount(<CardTitle title="Title" />)
  })

  it('Snapshot CardTitle', () => {
    const wrapper = mount(<CardTitle title="Title" />)
    expect(wrapper).toMatchSnapshot()
  })
})