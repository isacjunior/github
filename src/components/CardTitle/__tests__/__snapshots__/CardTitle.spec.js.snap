// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardTitle Snapshot CardTitle 1`] = `
<CardTitle
  title="Title"
>
  <styled.div>
    <div
      className="sc-bdVaJa kDsGQB"
    >
      <div>
        <h2>
          Title
        </h2>
        <div
          className="line-under"
        />
      </div>
    </div>
  </styled.div>
</CardTitle>
`;
